grr/server/grr_response_server/gui/ui/components/flow_args_form/osquery_query_helper/osquery_query_helper_test.ts

```diff
@@ -0,0 +1,61 @@
+import {newOsqueryTableSpec, newOsqueryColumnSpec} from './osquery_table_specs';
+import {QueryComposer} from './osquery_query_helper';
+
+fdescribe('QueryComposer', () => {
```

Comment:
Let's make sure we don't submit fdescribe accidentally.

Reply:
Good spot!